### PR TITLE
fix #9 update consolidate to 0.14.1 && rename jade to pug for warning

### LINKED ISSUE #9: npm WARN deprecated jade@1.11.0: Jade has been renamed to pug, please install the latest version of pug instead of jade

### PR DESCRIPTION
**Pull request by i5ting**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "co-render",
-  "version": "1.1.0",
```
Comment (haoxins) [inline package.json:3]: please don't bump version in a PR

File: package.json [hunk 2]
```diff
@@ -16,14 +16,14 @@
...
-    "consolidate": "^0.14.0",
+    "consolidate": "^0.14.1",
     "debug": "2"
   },
   "devDependencies": {
     "co": "4",
     "ejs": "2",
-    "jade": "1",
     "mocha": "2",
+    "pug": "^0.1.0",
```
Comment (haoxins) [inline package.json:26]: why `0.1.0` ? the latest is `2.0.0-alpha6`

File: index.js [hunk 3]
```diff
@@ -25,7 +25,8 @@ module.exports = render;
 
 function render(view, opts) {
   opts = opts || {};
-  var ext = opts.engine || extname(view).slice(1);
+  var _extname = extname(view).slice(1);
+  var ext = opts.engine || _extname === 'jade' ? 'pug' : _extname;
```
Comment (haoxins) [inline index.js:29]: For this case, I think the users should set `engine` opt to `pug`!

Comment (haoxins) [inline index.js:29]: If we need convert `jade` to `pug` automatically, it should be consolidate's logic.

File: package.json (hunk 2 above)
Comment (i5ting) [inline package.json:26]: see here https://github.com/tj/consolidate.js/blob/master/package.json#L37

File: index.js (hunk 3 above)
Comment (i5ting) [inline index.js:29]: co-render's logic depend on template file's extname , so if co-render doesn't process it, 

```
-  var ext = opts.engine || extname(view).slice(1);
```

this code will be wrong

File: package.json (hunk 1 above)
Comment (i5ting) [inline package.json:3]: ok

**Pull request closed without merging**